test/BUILD

```diff
@@ -214,6 +214,12 @@ scala_library(
     name = "MixJavaScalaLib",
     srcs = glob(["src/main/scala/scala/test/mix_java_scala/*.scala"]) +
            glob(["src/main/scala/scala/test/mix_java_scala/*.java"]),
+    javac_jvm_flags = [
```

Comment:
Was this added for demonstration/regression or is this needed?
If this is needed I'd love to understand why.
If this is for the former reason then it's also unclear to me how this protects from regression

Reply:
Thanks for catching this as well, this was not needed after all (I'd mistakenly thought it was).  I reverted the file.